hal/halx86/smp/ipi.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * PROJECT:     ReactOS Kernel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Source file for IPI management
```

Comment:
I would recommend to explain what the IPI abbreviation stands for, as it is not obvious for everyone.

Reply:
Inter-Processor Interrupts.